components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/VariantListController.java

```diff
@@ -86,7 +86,7 @@
 
     private static final String CONTROLLER_NAME = VARIANTS_STRING;
 
-    private static final String VARIANTS_ENABLING_FIELD_NAME = "genes";
+    private static final String VARIANTS_ENABLING_FIELD_NAME = "variants";
```

Comment:
Just want to make sure we do not merge this until we have the new UIX field? I think as discussed right now having enabling field to be `variants` does not work in some cases (e.g. consents don't play well with this and we may end up exporting variants when there is no consent for that?)

Reply:
This PR doesn't not affect any consent configurations or work and can be merged separately.